gym/lib/gym/options.rb

```diff
@@ -158,6 +158,10 @@ def self.plain_options
                                      env_name: "GYM_SDK",
                                      description: "The SDK that should be used for building the application",
                                      optional: true),
+        FastlaneCore::ConfigItem.new(key: :toolchain,
+                                     env_name: "GYM_TOOLCHAIN",
+                                     description: "The toolchain that should be used for building the application",
```

Comment:
@KrauseFx 

> Could you provide some more information of what a toolchain is?

From [Swift.org](https://swift.org/download/#using-downloads)

> An Xcode toolchain (.xctoolchain) includes a copy of the compiler, lldb, and other related tools needed to provide a cohesive development experience for working in a specific version of Swift.

Should I add this in description of gym options?

e.g.
```ruby
FastlaneCore::ConfigItem.new(key: :toolchain,
                             env_name: "GYM_TOOLCHAIN",
                             description: "The toolchain that should be used for building the application. An Xcode toolchain (.xctoolchain) includes a copy of the compiler, lldb, and other related tools needed to provide a cohesive development experience for working in a specific version of Swift",
                             optional: true),
```

> Is there a good way to validate the input, or to show some of the available values?

I have no idea. :cry:
Although we can get available value from `/Library/Developer/Toolchains/` and `/Applications/Xcode.app/Contents/Developer/Toolchains` by searching plist files, but it's not so simple and I think it's tough to validate it.